src/System.Drawing.Common/tests/Imaging/ImageAttributesTests.cs

```diff
@@ -0,0 +1,1503 @@
+// See the LICENSE file in the project root for more information.
+//
+// Copyright (C) 2005-2006 Novell, Inc (http://www.novell.com)
+//
+// Permission is hereby granted, free of charge, to any person obtaining
+// a copy of this software and associated documentation files (the
+// "Software"), to deal in the Software without restriction, including
+// without limitation the rights to use, copy, modify, merge, publish,
+// distribute, sublicense, and/or sell copies of the Software, and to
+// permit persons to whom the Software is furnished to do so, subject to
+// the following conditions:
+//
+// The above copyright notice and this permission notice shall be
+// included in all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+// EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+// MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+// NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE
+// LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+// OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
+// WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+//
+
+using System.Collections.Generic;
+using System.Drawing.Drawing2D;
+using System.Drawing.Tests;
+using System.IO;
+using Xunit;
+
+namespace System.Drawing.Imaging.Tests
+{
+    public class ImageAttributesTests
+    {
+        private readonly Rectangle _rectangle = new Rectangle(0, 0, 64, 64);
+        private readonly Color _actualYellow = Color.FromArgb(255, 255, 255, 0);
+        private readonly Color _actualGreen = Color.FromArgb(255, 0, 255, 0);
+        private readonly Color _expectedRed = Color.FromArgb(255, 255, 0, 0);
+        private readonly Color _expectedBlack = Color.FromArgb(255, 0, 0, 0);
+        private readonly ColorMatrix _colorMatrix = new ColorMatrix(new float[][]
+        {
+            new float[] {1, 0, 0, 0, 0},
+            new float[] {0, 0, 0, 0, 0},
+            new float[] {0, 0, 1, 0, 0},
+            new float[] {0, 0, 0, 1, 0},
+            new float[] {0, 0, 0, 0, 0},
+        });
+
+        private readonly ColorMatrix _grayMatrix = new ColorMatrix(new float[][] {
+            new float[] {1, 0, 0, 0, 0},
+            new float[] {0, 2, 0, 0, 0},
+            new float[] {0, 0, 3, 0, 0},
+            new float[] {0, 0, 0, 1, 0},
+            new float[] {0, 0, 0, 0, 0},
+        });
+
+        private readonly ColorMap[] _colorMap = new ColorMap[]
+        {
+            new ColorMap() { OldColor = Color.FromArgb(255, 255, 255, 0), NewColor = Color.FromArgb(255, 255, 0, 0) }
+        };
+
+        [ConditionalFact(nameof(PlatformDetection) + "." + nameof(PlatformDetection.IsNotWindowsNanoServer))]
+        public void Ctor_Default_Success()
+        {
+            using (var imageAttr = new ImageAttributes())
+            {
+                Assert.NotEqual(null, imageAttr);
```

Comment:
This isn't really a useful assertion, because `imageAttr` cannot be null after the constructor call succeeds. You can probably just remove the assertion.